tests/conftest.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+from supergood import Client
+from tests.helper import get_config, get_remote_config
+
+
+@pytest.fixture(scope="session")
```

Comment:
The http server is mocked at the session level. I was having some trouble with implementing this as a function or class level mock with each mocked client talking to the same http server.

Continuing to figure out if there's a better way, but for now this works well enough albeit it's still a little clunky. Ideally each test function / test class can prepare it's client.